tests/multigrid_invert_test.cpp

```diff
@@ -377,6 +377,9 @@ void setMultigridParam(QudaMultigridParam &mg_param)
   mg_param.run_low_mode_check = low_mode_check ? QUDA_BOOLEAN_YES : QUDA_BOOLEAN_NO;
   mg_param.run_oblique_proj_check = oblique_proj_check ? QUDA_BOOLEAN_YES : QUDA_BOOLEAN_NO;
 
+  // Is NOT a staggered solve
+  mg_param.is_staggered = QUDA_BOOLEAN_NO;
```

Comment:
Having to set this is an interface change.  I also just realized that you haven't set `is_staggered` in check_params.h.  Probably best to default to `QUDA_BOOLEAN_NO` so that all codes that use QUDA MG for Wilson/clover/twisted-mass fermions don't suddenly break.

Reply:
Done, I think correctly.